src/optimize/dynamic_dll_plugin/dll_config_model.js

```diff
@@ -83,6 +83,15 @@ function generateDLL(config) {
               include: /[\/\\]node_modules[\/\\]normalize-url[\/\\]/,
               exclude: /[\/\\]node_modules[\/\\]normalize-url[\/\\](.+?[\/\\])*node_modules[\/\\]/,
             },
+            // TODO: remove when we drop support for IE11
+            // We need because query-string is distributed without
+            // any kind of transpilation
+            // More info: https://github.com/elastic/kibana/issues/58684
+            {
+              test: /\.js$/,
+              include: /[\/\\]node_modules[\/\\]query-string[\/\\]/,
+              exclude: /[\/\\]node_modules[\/\\]query-string[\/\\](.+?[\/\\])*node_modules[\/\\]/,
```

Comment:
Excluding dependencies of query-string will not work for us because the dependencies are also not compiled.

Reply:
Agree, I removed `exclude`